hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulator.java

```diff
@@ -6,8 +6,9 @@
 import com.hazelcast.spi.Operation;
 
 import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.hazelcast.client.impl.protocol.util.BitUtil.CACHE_LINE_LENGTH;
```

Comment:
We already have two distinct classes; `com.hazelcast.nio.Bits` for bit/byte/primitive manipulation and `com.hazelcast.util.QuickMath` for shorthand math calculations. I think we should request client team to remove `com.hazelcast.client.impl.protocol.util.BitUtil` and add required methods/constants to these two classes.